service_discovery/proto/build.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    tonic_build::compile_protos("../proto/chariott/v1/chariott_registry.proto")?;
```

Comment:
same here in terms of using the code name of the project

Reply:
I have removed it from all new code, kept it in the existing package name, as well as comments describing the files. Please let me know what you think